src/App.tsx

```diff
@@ -86,12 +103,7 @@ export const App = () => {
               data-cy="titleInput"
               placeholder="Enter a title"
               value={inputText}
-              onChange={(event) => {
-                setSubmittedTitle(true);
-                const char = event.target.value;
-
-                setInputText(char.replace(/[^\p{L} ' ']/gu, ''));
-              }}
+              onChange={(event) => handleInputOnChange(event.target.value)}
```

Comment:
```suggestion
              onChange={handleInputOnChange}
```
Process event as argument of your handlers. Here and in other cases. Even if it's one line of code.